src/components/DRB/DRBContainer.tsx

```diff
@@ -39,38 +42,68 @@ const DRBContainer = ({
       <CardHeading size="h6" id="drb-sidebar-container-heading">
-        Results from Digital Research Books Beta
+        {hasResults
+          ? "Results from Digital Research Books Beta"
+          : "No results found from Digital Research Books Beta"}
       </CardHeading>
       <CardContent id="drb-sidebar-container-content">
-        <Text size="body2">
-          Digital books for research from multiple sources world wide- all free
-          to read, download, and keep. No Library Card is Required.{" "}
-          <ExternalLink href={appConfig.urls.drbAbout}>
-            Read more about the project
-          </ExternalLink>
-          .
-        </Text>
-        <SimpleGrid columns={1} gap="grid.s" pb="s">
-          {drbResults.map((result: DRBResult) => (
-            <DRBCard key={result.id} drbResult={result} />
-          ))}
-        </SimpleGrid>
-        {totalWorks && (
-          <ExternalLink
-            href={`${DRB_BASE_URL}/search${drbQuery}`}
-            fontSize={{
-              base: "mobile.body.body2",
-              md: "desktop.body.body2",
-            }}
-            type="standalone"
-            fontWeight="bold"
-          >
-            <>
-              View {totalWorks === 1 ? "" : "all"} {totalWorks.toLocaleString()}{" "}
-              result
-              {totalWorks === 1 ? "" : "s"} in Digital Research Books Beta
-            </>
-          </ExternalLink>
+        <Box mb="s">
+          <Text size="body2" mb="xxs">
+            Digital books for research from multiple sources worldwide - all
+            free to read, download, and keep. No library card required.
+          </Text>
+          <Text size="body2">
+            <ExternalLink href={appConfig.urls.drbAbout}>
+              Read more about the project
+            </ExternalLink>
+          </Text>
+        </Box>
+
+        {hasResults ? (
+          <>
+            <SimpleGrid columns={1} gap="grid.s" pb="s">
+              {drbResults.map((result: DRBResult) => (
+                <DRBCard key={result.id} drbResult={result} />
+              ))}
+            </SimpleGrid>
+            <ExternalLink
+              href={`${DRB_BASE_URL}/search${drbQuery}`}
+              fontSize={{
+                base: "mobile.body.body2",
+                md: "desktop.body.body2",
+              }}
+              type="standalone"
+              fontWeight="bold"
+            >
+              <>
+                View {totalWorks === 1 ? "" : "all"}{" "}
+                {totalWorks.toLocaleString()} result
+                {totalWorks === 1 ? "" : "s"} in Digital Research Books Beta
+              </>
+            </ExternalLink>
+          </>
+        ) : (
+          <>
+            <ExternalLink
+              href={appConfig.urls.drbAbout}
+              aria-label="Explore Digital Research Books Beta"
+            >
+              <Box mb="xs" bg="ui.white" p="s" borderRadius="5px">
+                <Image src={drbPromoImage} alt="" />
```

Comment:
Importing the image into the component and then passing it as a value feels wrong. If you put the image file in the `public` directory, then you should be able to put the URL directly here.

Reply:
The Next JS docs say that the Image component can take a statically loaded asset. It can also take an absolute path, but it'll change based on the environment and using a relative path requires the loader prop:

https://nextjs.org/docs/pages/api-reference/components/image

Reply:
We discussed this offline: we'll keep this as recommended by Next for local images, including the usage of Next's `Image` component.